autoload/airline/extensions/tabline/ctrlspace.vim

```diff
@@ -71,7 +71,11 @@ function! airline#extensions#tabline#ctrlspace#add_tab_section(builder, pull_rig
           \ .s:highlight_groups[(4 * tab.modified) + (3 * tab.current)]
           \ .pos_extension
 
-    call a:builder.add_section_spaced(group, '%'.tab.index.'T'.tab.title.ctrlspace#api#TabBuffersNumber(tab.index).'%T')
+    if get(g:, 'airline#extensions#tabline#tabs_numbered', 0) == 0
```

Comment:
Is this variable documented?

Reply:
No, but I can add documentation for it.  Can you point me to where I should do that?

Reply:
somewhere in doc/vim-airline.txt somwhere were similar config options for the tabline are documented

Reply:
Documentation added